conans/client/migrations.py

```diff
@@ -122,6 +125,27 @@ def _get_prefs(layout):
     return [PackageReference(layout.ref, s) for s in folders]
 
 
+def migrate_tgz_location(cache, out):
+    """ In Conan 1.31, the temporary .tgz are no longer stored in the content folders. In case
+    they are found there, they can be removed, and the next time they are needed (upload), they
+    will be compressed again
+    """
+    out.info("Removing temporary .tgz files, they are stored in a different location now")
+    refs = _get_refs(cache)
+    for ref in refs:
+        try:
+            base_folder = os.path.normpath(os.path.join(cache.store, ref.dir_repr()))
+            for d, _, fs in os.walk(base_folder):
+                for f in fs:
+                    if f in (EXPORT_SOURCES_TGZ_NAME, EXPORT_TGZ_NAME, PACKAGE_TGZ_NAME):
```

Comment:
Not yet, it will go in the other PR

```suggestion
                    if f in (PACKAGE_TGZ_NAME, ):
```